src/index.js

```diff
@@ -24,19 +24,17 @@ module.exports = {
     'block-spacing': 'off',
     'brace-style': 'off',
     camelcase: 'off',
-    'comma-dangle': 'off',
```

Comment:
Some rules, like this one, don't seem to have an addition changing from `off` to `error`. Any reason for that?